exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,11 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole) {
```

Comment:
A non-empty string will always evaluate to `true` -- so this will always return "Hi, I'm Daniel, I'm a mentor", unless `danielsRole` is empty.

What you actually want is `if (danielsRole === "mentor")`.

Reply:
Changes made.